.eslintrc.json

```diff
@@ -15,7 +15,9 @@
   },
 
   "plugins": [],
-  "extends": [],
+  "extends": [
+    "plugin:import/typescript"
```

Comment:
Do we have lint error in this file too?

Reply:
I removed the changes and it is working now without the changes to this file.